File: monopoly.client/src/app/models/cell.model.ts
export interface ICell {
    id: string;
    label: string;
    price?: number | null;
    color?: string | null;
    type: CellType
}

export enum CellType {
    Start,
    Treasury,
    IncomeTax,
    Railway,
    Chance,
    Arrested,
    Powerhouse,
    Parking,
    WaterSupply,
    Jail,
    GreyStreet,
    PinkStreet,
    YellowStreet,
    GreenStreet,
    BlueStreet,
    CornStreet,
    OrangeStreet,
    RedStreet
}

File: monopoly.Server/Models/CLient/CellModel.cs
using monopoly.Server.Models.Backend;

namespace monopoly.Server.Models.Client
{
    public class CellModel(string id, string label, CellType type, float? price = null, string? color = null)
    {
        public string Id { get; set; } = id;
        public string Label { get; set; } = label;
        public float? Price { get; set; } = price;
        public string? Color { get; set; } = color;
        public CellType Type { get; set; } = type;
    }
}
